executor/runner/runner.go

```diff
@@ -72,16 +73,17 @@ func New(ctx context.Context, m metrics.Reporter, logUploaders *uploader.Uploade
 	dockerRuntime := func(ctx context.Context, cfg config.Config) (runtimeTypes.Runtime, error) {
 		return docker.NewDockerRuntime(ctx, m, dockerCfg, cfg)
 	}
-	return WithRuntime(ctx, m, dockerRuntime, logUploaders, cfg)
+	return WithRuntime(ctx, m, dockerRuntime, logUploaders, cfg, &dockerCfg)
```

Comment:
So, `dockerCfg` is supposed be Docker specific. Whereas WithRuntime, or runner.go is supposed to be container runtime agnostic.